tests/executors/test_celery_executor.py

```diff
@@ -75,6 +75,17 @@ def _prepare_app(broker_url=None, execute=None):
     patch_app = mock.patch('airflow.executors.celery_executor.app', test_app)
     patch_execute = mock.patch('airflow.executors.celery_executor.execute_command', test_execute)
 
+    backend = test_app.backend
+
+    if hasattr(backend, 'ResultSession'):
```

Comment:
Is this check necessary?

Reply:
Yes. We can also keep this data in non-DatabaaseBackend.
https://github.com/celery/celery/tree/master/celery/backends

Reply:
Yeah, exactly. We only currently test DB backends in our tests, but I felt better to future proof this for the sake of a single `if`.